packages/notifications/src/elements/global-alert/GlobalAlert.tsx

```diff
@@ -33,16 +36,24 @@ const GlobalAlertComponent = forwardRef<HTMLDivElement, IGlobalAlertProps>(
       warning: <WarningIcon />,
       info: <InfoIcon />
     }[type];
+    const theme = useContext(ThemeContext) || DEFAULT_THEME;
+    /* [1] */
+    const lightTheme: DefaultTheme = useMemo(
+      () => ({ ...theme, colors: { ...theme.colors, base: 'light' } }),
+      [theme]
+    );
 
     return (
-      <GlobalAlertContext.Provider value={useMemo(() => ({ type }), [type])}>
-        {/* [1] */}
-        {/* eslint-disable-next-line jsx-a11y/prefer-tag-over-role */}
-        <StyledGlobalAlert ref={ref} role="status" $alertType={type} {...props}>
-          <StyledGlobalAlertIcon $alertType={type}>{icon}</StyledGlobalAlertIcon>
-          {props.children}
-        </StyledGlobalAlert>
-      </GlobalAlertContext.Provider>
+      <ThemeProvider theme={lightTheme}>
```

Comment:
Fun fact: the `theme` prop of nested `ThemeProviders` have access to the parent theme. 😉 

https://styled-components.com/docs/api#themeprovider

```suggestion
      <ThemeProvider theme={theme => ({...theme, colors: { ...theme.colors, base: 'light' })}>
```

Reply:
Hey, I completely forgot about that!